### ISSUE TITLE
Compiler error in .fs file when open referenced library - example with Suave template

### ISSUE THREAD

**Issue by EdouardM** [ionide/ionide-vscode-fsharp#171]
Hello, 

I am using VS Code with Ionide a lot to code small 1 file programs. The add-in is great !
I would like to build something bigger with Suave and .fs files.  

## Steps to reproduce the issue 
* In a new empty directoy I run the command "F# New project" 
* choose the "suave" template 
* create a new file App.fs and copy the code in the default .fsx coming with the template
* add App.fs file to the project with the commande "F#: Add current file to project"

##Issue 
In the App.fs file I have compiler errors telling "Suave" library is not referenced whereas it has been installed via paket and it is referenced in .fsproj file. 

Please refer to attached screenshot.
![2016-09-27_08h12_12](https://cloud.githubusercontent.com/assets/7335389/18892439/ccc5eb88-8509-11e6-9aab-f8fa47f29c94.png)

## Expected behaviour
I would expect to reference the library and have compiler not complaining with intellisense working.

Thanks for your help,
Edouard

**Comment by Krzysztof-Cieslak** [comment]
1. Check if `.fs` is for sure added to the `.fsproj` file, if that's not the case check Forge output (choose Forge from the list in output panel )

2. Try using project name without special characters in it.

3. Try using whole path without special characters in it (I've noticed that you have `01.Dev` somewhere in the full path to file. )

**Comment by EdouardM** [comment]
Hello Krzysztof, 

I created another project without special character in whole path and project name. I also checked the file is in fsproj file.

Still the compiler does not find the library. 

Check below: 
![2016-09-28_error_referencing_library_fsfile_special_char_removed](https://cloud.githubusercontent.com/assets/7335389/18902352/ba5165f6-8552-11e6-8e7d-e81e9281f95a.png)
![2016-09-28_error_referencing_library_fsfile_file_included_fsproj](https://cloud.githubusercontent.com/assets/7335389/18902355/bfdea0d8-8552-11e6-9027-6a5c6079c708.png)

Thanks for you help, 
Edouard

**Comment by EdouardM** [comment]
Hello Krzysztof,

I tried to solve the issue by leaving Visual studio and reinstalling everything but the issue persists. 

I noticed that the F# language service is only retrieving one reference when reading my .fsproj file. It retrieves only FSharp.Core.dll whereas there are several reference in .fsproj file. 

In this screenshot there is only one reference in the list
https://cloud.githubusercontent.com/assets/7335389/18902352/ba5165f6-8552-11e6-8e7d-e81e9281f95a.png

I try to modify the .fsproj file manually without success. 

Please advise
Edouard

**Comment by cloudRoutine** [comment]
@EdouardM i just created a new project from the suave template and everything is working fine

![](http://i.imgur.com/Qwf1CTW.png)

Screenshots are not enough for us to help you diagnose this issue. Could you either upload a minimal reproduction as a repo on github or use `git archive -o repro.zip HEAD` and drop the file into a comment on this issue.

**Comment by EdouardM** [comment]
Hello @cloudRoutine 

I uploaded a repo to illustrate the issue. I only used the template suave. I have compiler errors in vscode whereas the project builds fine. 

The repo is available here:  https://github.com/EdouardM/musicStore

I did some investigation in the logic of MS Build file in order to inspect some variable. I turned out I had: 
$(Configuration) = ''
$(Platform) = 'MCD' 

This is because I am using an HP PC.  I modified the .fsproj file to set the variable Platform to 'AnyCPU'. 

Now the compiler errors are gone, I have the same situation as you at last !  

Maybe it would be nice to add a special case to handle this problem in the template. 

Kind regards;
Edouard
